configs/defs/generic_linux.textproto

```diff
@@ -9602,8 +9602,6 @@ benchmark_configs: {
       "      }"
       "    }"
       "  }"
-      "}"
-      "image_scanning:{"
       "  check_alternatives:{"
```

Comment:
Remove this check_alternative too since it's a duplicate of the above check_alternative